deltacat/tests/compute/test_compact_partition_incremental.py

```diff
@@ -204,6 +211,8 @@ def test_compact_partition_incremental(
     drop_duplicates_param: bool,
     read_kwargs_provider_param: Any,
     skip_enabled_compact_partition_drivers,
+    assert_compaction_audit: Optional[Callable],
+    compactor_version: Optional[CompactorVersion],
     is_inplace: bool,
```

Comment:
Not used anywhere in this method?

Reply:
If I don't include these parameters, I get an error when calling pytest.mark.parametrize. I think they are needed here even though the linter grays them out. For example, the `test_name` parameter is also not used in the method `test_compact_partition_incremental` but is needed during parametrizing. I intended to only add these new arguments to `rebase_then_incremental`, but because I added `assert_compaction_audit` to the `BaseCompactorTestCase` for simplicity and `compactor_version` during enriching (all three test case files use the same function to enrich), `test_compact_partition_incremental` and `test_compact_partition_rebase` need the args as well.

Reply:
Yeah, you can actually add the assertion here.